.golangci.yml

```diff
@@ -96,6 +96,7 @@ linters:
     - staticcheck
     - unused
     - whitespace
+    - revive
```

Comment:
Please keep the list sorted.

Also, please add revive's configuration above

Reply:
> Oh, and it should be added to one block and removed from another block in `.golangci-new.yml` too

added.

> Also, please add revive's configuration above

added.

Reply:
It was removed from one block, but wasn't added into another

Reply:
> It was removed from one block, but wasn't added into another

yes, it was not clear that we manage different lists of disabled linters.